Sources/Rendering/Core/RenderWindowInteractor/index.js

```diff
@@ -927,6 +930,18 @@ function vtkRenderWindowInteractor(publicAPI, model) {
       }
     }
   };
+
+  // Stop animating if the renderWindowInteractor is deleted.
+  const superDelete = publicAPI.delete;
+  publicAPI.delete = () => {
+    if (animationRequesters) {
+      animationRequesters.forEach((animationRequester) =>
```

Comment:
I find it a bit risky to write it that way  because cancelAnimation modifies animationRequesters.
`while(animationRequesters.length)` seems safer to me.